content/git/concepts/revert/revert.md

```diff
@@ -0,0 +1,67 @@
+---
+Title: 'Revert'
+Description: 'The git revert command is used to create a new commit that will undo any changes made to a previous commit.'
+Subjects:
+  - 'Bash/Shell'
+  - 'Developer Tools'
+Tags:
+  - 'Git'
+  - 'Version Control'
+CatalogContent:
+  - 'learn-the-command-line'
+  - 'learn-git'
+---
+
+In Git, the **`revert`** command is used to create a new commit that will undo any changes made to a previous commit. This can be considered as the safe undo command because it keeps previous history. The command is best used when working with others and need to preserve the commit history.
+
+> **Note:** This command can be mistaken for the [`reset`](https://www.codecademy.com/resources/docs/git/reset) command, which is a more dangerous undo command because it alters existing hitory.
+
+## Syntax
+
+```shell
+git revert <commit-reference>
+```
+
+### Commit Reference
```

Comment:
We don't need this header. Delete it so the following lines are in the syntax section. Make sure to also delete the extra blank line so there's only one blank line between the code block and the first sentence.